packages/excalidraw/components/App.tsx

```diff
@@ -3834,7 +3836,7 @@ class App extends React.Component<AppProps, AppState> {
             y: element.y + offsetY,
           });
 
-          updateBoundElements(element, {
+          updateBoundElements(element, this.scene.getNonDeletedElementsMap(), {
```

Comment:
currently I am seeing a mix of deleted and non deleted maps when using `elementsMap`, however passing non deleted map should ideally work,
 @dwelle do you see any concern with passing just non deleted elements map ?